sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/plan/strategy/ParallelStrategyTest.java

```diff
@@ -55,45 +60,16 @@ public void testParallelExecution() {
         Assert.assertEquals(3, getCandidates().size());
 
         when(el0.isComplete()).thenReturn(true);
-        when(el0.isPending()).thenReturn(false);
+        when(el0.isEligible(any(Collection.class))).thenReturn(false);
         Assert.assertEquals(2, getCandidates().size());
 
         when(el1.isComplete()).thenReturn(true);
-        when(el1.isPending()).thenReturn(false);
-        Assert.assertEquals(1, getCandidates().size());
-        Assert.assertEquals(el2, getCandidates().iterator().next());
-
-        when(el2.isComplete()).thenReturn(true);
-        when(el2.isPending()).thenReturn(false);
-        Assert.assertTrue(getCandidates().isEmpty());
-    }
-
-    @Test
-    public void testDirtyAssetAvoidance() {
```

Comment:
The dirtiness logic is in the Step now, which is mocked out.  So it doesn't belong here, but I suppose I should add the test back into StepTest.